src/http/request_serializer.hpp

```diff
@@ -39,15 +40,32 @@ class RequestSerializer {
         if (settings.find("http/url") == settings.end()) {
             throw MissingUrlError();
         }
+<<<<<<< HEAD
+        url = parse_url(settings.at("url"));
+        protocol = settings.get("http_version", std::string("HTTP/1.1"));
+        method = settings.get("method", std::string("GET"));
+        path = settings.get("path", std::string(""));
+        if (path != "" && path[0] != '/') {
+            path = "/" + path;
+        }
+
+=======
         url = parse_url(settings.at("http/url"));
         protocol = settings.get("http/http_version", std::string("HTTP/1.1"));
         method = settings.get("http/method", std::string("GET"));
+>>>>>>> master
```

Comment:
@helloimcarmine you should manually solve the conflict here by choosing which version of the two you prefer or cherry picking the right lines from the two.